packages/autorest.typescript/test/smoke/generated/arm-package-deploymentscripts-2019-10-preview/package.json

```diff
@@ -21,17 +21,17 @@
   "types": "./types/arm-package-deploymentscripts-2019-10-preview.d.ts",
   "devDependencies": {
     "@microsoft/api-extractor": "^7.31.1",
+    "mkdirp": "^2.1.2",
+    "typescript": "~5.0.0",
+    "uglify-js": "^3.4.9",
+    "rimraf": "^5.0.0",
+    "dotenv": "^16.0.0",
     "@rollup/plugin-commonjs": "^24.0.0",
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-multi-entry": "^6.0.0",
     "@rollup/plugin-node-resolve": "^13.1.3",
-    "mkdirp": "^2.1.2",
     "rollup": "^2.66.1",
-    "rollup-plugin-sourcemaps": "^0.6.3",
-    "typescript": "~5.0.0",
-    "uglify-js": "^3.4.9",
-    "rimraf": "^5.0.0",
-    "dotenv": "^16.0.0"
+    "rollup-plugin-sourcemaps": "^0.6.3"
```

Comment:
> The smoke tests are failing because they do npm install and npm build, however, installation fails to get @azure/dev-tool. Any suggestion to fix these? These tests have the AzureSdkForJs option.

Is the issue existing? I assume in non js-repo there will have no real change just order change.

Reply:
Prior to this change it doesn't matter because all generated use `rollup` so works fine with npm install.  Now package generated for js repo would failed to install. the old `AzureSdkForJs` option (`--AzureSdkForJs=false`) really is NotAzureSdkForJs so I renamed it and added it for all smoke tests.